README.md

```diff
@@ -97,24 +126,39 @@ You can deploy this project using:
 - LinkedIn: [Ngala Mac](https://www.linkedin.com/in/...)
 
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 ## 🔭 Future Features <a name="future-features"></a>
 
-- A more improved user interface
-- user dashboard
+- [ ] **Adding more pages**
+- [ ] **Adding responsiveness**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
+Feel free to create an issue on this repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 ## ⭐️ Show your support <a name="support"></a>
 
-If you like this project you can go ahead give a star to the repository.
+If you like this project give me a star.
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I give thanks to Microverse community for investing in me. I am looking forth to building more projects
```

Comment:
- [ ] 👋 Hey there! Great work on your project! 🚀, it's important to give credit to the original author in your README. 🎨
Not only is this a requirement under the license, but it's also a way to show appreciation for the artist's hard work! 🙌 Keep up the good work! 😊